integration-cli/docker_cli_events_test.go

```diff
@@ -615,45 +608,43 @@ func (s *DockerCLIEventSuite) TestEventsFilterType(c *testing.T) {
 func (s *DockerCLIEventSuite) TestEventsSpecialFiltersWithExecCreate(c *testing.T) {
 	since := daemonUnixTime(c)
 	runSleepingContainer(c, "--name", "test-container", "-d")
-	waitRun("test-container")
+	cli.WaitRun(c, "test-container")
 
-	dockerCmd(c, "exec", "test-container", "echo", "hello-world")
+	cli.DockerCmd(c, "exec", "test-container", "echo", "hello-world")
 
-	out, _ := dockerCmd(
-		c,
+	out := cli.DockerCmd(c,
 		"events",
 		"--since", since,
 		"--until", daemonUnixTime(c),
 		"--filter",
 		"event='exec_create: echo hello-world'",
-	)
+	).Stdout()
 
 	events := strings.Split(strings.TrimSpace(out), "\n")
 	assert.Equal(c, len(events), 1, out)
 
-	out, _ = dockerCmd(
-		c,
+	out = cli.DockerCmd(c,
 		"events",
 		"--since", since,
 		"--until", daemonUnixTime(c),
 		"--filter",
 		"event=exec_create",
-	)
+	).Stdout()
 	assert.Equal(c, len(events), 1, out)
 }
 
 func (s *DockerCLIEventSuite) TestEventsFilterImageInContainerAction(c *testing.T) {
 	since := daemonUnixTime(c)
-	dockerCmd(c, "run", "--name", "test-container", "-d", "busybox", "true")
-	waitRun("test-container")
+	cli.DockerCmd(c, "run", "--name", "test-container", "-d", "busybox", "true")
+	cli.WaitRun(c, "test-container")
```

Comment:
Ah, looks like this is one of the failures. The old code was actually using `waitRun("test-container")` incorrectly, as it didn't check the result (so it could fail).

The new code checks if the container is running, but apparently it may have depended on the old behavior?